tdrs-frontend/.snyk

```diff
@@ -0,0 +1,75 @@
+version: v1.14.1
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JS-NODESASS-535497:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-535498:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-535502:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540956:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540958:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540964:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540974:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540978:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540990:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540992:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540994:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540996:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-541000:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-541002:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:10:10.191Z'
+  SNYK-JS-NODESASS-540980:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:14:37.144Z'
+  SNYK-JS-NODESASS-540998:
+    - node-sass:
+        reason: This is only used in compilation and wont be part for deployment.
+        expires: '2020-09-27T00:14:37.145Z'
+  SNYK-JS-NODEFORGE-598677:
+    - react-scripts > webpack-dev-server > selfsigned > node-forge:
```

Comment:
selfsigned probably isn't used since we're serving on cloud.gov. you might want to update the policy to note that.